docs/api/app.md

```diff
@@ -1235,7 +1239,28 @@ the user to confirm the operation, you may do so using the
 move to fail. For instance if the user cancels the authorization dialog, this
 method returns false. If we fail to perform the copy, then this method will
 throw an error. The message in the error should be informative and tell
-you exactly what went wrong
+you exactly what went wrong.
+
+By default, if an app of the same name as the one being moved exists in the Applications directory and is _not_ running, the existing app will be trashed and the active app moved into its place. If it _is_ running, the pre-existing running app will assume focus and the the previously active app will quit itself. This behavior can be changed by invoking the optional callback handler, where the boolean returned by the handler determines whether or not the move conflict is resolved with default behavior.
+
+For example:
+
+```js
+app.moveToApplicationsFolder({
+  conflictHandler: conflictType => {
+    if (conflictType === 'exists') {
+      dialog.showMessageBoxSync({
```

Comment:
This won't work if I'm reading this right, need to do `return dialog.showMessageBoxSync(...) === 1`, sync doesn't have a callback

Reply:
oh derp yes